contoso-bicycle/README.md

```diff
@@ -0,0 +1,115 @@
+
+### Reliability
+
+- Global presence: The customer base is focused on the West Coast of North America.
+
+- Business continuity: The workloads need to be highly available at a minimum cost. They have a Recovery Time Objective (RTO) of 4 hours.
+
+- On-premises connectivity: They don’t need to connect to on-premises data centers or legacy applications.
+
+### Performance efficiency
+
+The web service’s host should have these capabilities.
+
+- Auto scaling: Automatically scale to handle the demands of expected traffic patterns. The web service is unlikely to experience a high-volume scale event. The scaling methods shouldn't drive up the cost.
+
+- Right sizing: Select hardware size and features that are suited for the web service and are cost effective.
+
+- Growth: Ability to expand the workload or add adjacent workloads as the product matures and gains market adoption.
+
+- Monitoring: Emit telemetry metrics to get insights into the performance and scaling operations. Integration with Azure Monitor is preferred.
+
+- Workload-based scaling: Allow granular scaling per workload and independent scaling between different partitions in the workload.
+
+### Security
+
+- Identity management: Contoso is an existing Microsoft 365 user. They rely heavily on Azure Active Directory as their control plane for identity.
+
+- Certificate: They must expose all web services through SSL and aim for end-to-end encryption, as much as possible.
+
+- Network: They have existing workloads running in Azure Virtual Networks. They would like to minimize direct exposure to Azure resources to the public
+internet. Their existing architecture runs with regional hub and spoke topologies. This way, the network can be expanded in the future and also provide workload isolation.
+All web applications require a web application firewall (WAF) service to help govern HTTP traffic flows.
+
+- Secrets management: They would like to use a secure store for sensitive information.
+
+- Container registry: Currently not using a registry and are looking for guidance.
+
+- Container scanning: They know the importance of container scanning but are concerned about added cost. The information isn't sensitive, but would like the option to scan in the future.
+
+### Operational excellence
+
+- Logging, monitoring, metrics, alerting: They use Azure Monitor for their existing workloads. They would like to use it for AKS, if possible.
+
+- Automated deployments: They understand the importance of automation. They build automated processes for all infrastructure so that environments and workloads
+can easily be recreated consistently and at any time.
+
+### Cost optimization
+
+- Cost center: There’s only one line-of-business. So, all costs are billed to a single cost center.
+
+- Budget and alerts: They have certain planned budgets. They want to be alerted when certain thresholds like 50%, 75%, and 90% of the plan has been reached.
+
+## Design and technology choices

```

Comment:
have we considered moving this to main README where we list all the components/technologies in this RI?

Reply:
I had not considered that.  I was trying to salvage this before the next docs publish (in about two hours), didn't try to merge it in.  Just took mostly "as is" for now.